packages/fast-form-generator-react/src/form/form-category.style.ts

```diff
@@ -1,49 +1,77 @@
-import { toPx } from "@microsoft/fast-jss-utilities";
-import { applyHeaderStyle } from "../utilities/form-input.style";
+import { background200, background800, foreground300 } from "../utilities";
 import { ComponentStyles, CSSRules } from "@microsoft/fast-jss-manager";
 import { FormCategoryClassNameContract } from "../class-name-contracts/";
-import formCategory from "./form-category";
-
-// tslint:disable-next-line
-const dropdownArrow: string =
-    "PHN2ZyBpZD0iTGF5ZXJfMSIgZGF0YS1uYW1lPSJMYXllciAxIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAzMiAzMiI+PHRpdGxlPmNoZXZyb24tZG93bjwvdGl0bGU+PHBhdGggZD0iTTMwLjUsNy4yOSwzMS45LDguNywxNi4yLDI0LjQuNSw4LjcsMS45LDcuMjlsMTQuMywxNC4zWiIvPjwvc3ZnPg==";
 
 const styles: ComponentStyles<FormCategoryClassNameContract, {}> = {
     formCategory_button: {
-        margin: `${toPx(20)} 0 0 0`,
-        minHeight: toPx(40),
-        fontWeight: "bold",
-        background: "none",
+        minHeight: "30px",
+        boxSizing: "border-box",
         outline: "0",
+        fontWeight: "600",
+        color: foreground300,
+        textTransform: "uppercase",
         border: "none",
+        background: background200,
+        borderTop: `1px solid ${background800}`,
+        borderBottom: `1px solid ${background800}`,
         position: "relative",
-        width: "100%",
+        width: "calc(100% + 10px)",
         fontFamily: "inherit",
-        fontSize: toPx(14),
-        padding: `${toPx(10)} 0`,
+        fontSize: "11px",
+        padding: "0 0 0 10px",
+        marginLeft: "-10px",

```

Comment:
negative margin is typically a code smell. I think the issue here is on line 18 where you are setting "width". I think you want,

`calc(100% - 10px)`;

With this, you should also be able to remove the padding here.